### ISSUE TITLE
pdm removes conda

### ISSUE THREAD

**Issue by Firionus** [pdm-project/pdm#2409]
- [x] I have searched the issue tracker and believe that this is not a duplicate.

**Make sure you run commands with `-v` flag before pasting the output.**

## Steps to reproduce

- Use pdm to manage the packages in a conda environment (use a conda interpreter)
- add a global package
- remove the global package

<details>
  <summary>Dockerfile</summary>

```
FROM continuumio/miniconda3:23.9.0-0
RUN apt-get update
RUN apt-get install -y curl
RUN curl -sSL https://pdm-project.org/install-pdm.py | python3 -
RUN echo "export PATH=/root/.local/bin:$PATH" >> ~/.bashrc
```
</details>

<details>
  <summary>Console Dump from Docker Container</summary>

```
(base) root@6dda1e96f2c9:/# conda --version
conda 23.9.0
(base) root@6dda1e96f2c9:/# pdm add -gv cowsay
Changes are written to pyproject.toml.
Adding packages to default dependencies: cowsay
STATUS: Resolving dependencies
pdm.termui: ======== Start resolving requirements ========
pdm.termui:   pip
pdm.termui:   setuptools
pdm.termui:   wheel
pdm.termui:   cowsay
pdm.termui:   python>=3.11.5,<3.11.6
pdm.termui:   Adding requirement pip
pdm.termui:   Adding requirement setuptools
pdm.termui:   Adding requirement wheel
pdm.termui:   Adding requirement cowsay
pdm.termui:   Adding requirement python>=3.11.5,<3.11.6
pdm.termui: ======== Starting round 0 ========
STATUS: Resolving: new pin python>=3.11.5,<3.11.6
pdm.termui: Pinning: python None
pdm.termui: ======== Ending round 0 ========
pdm.termui: ======== Starting round 1 ========
STATUS: Resolving: new pin cowsay 6.1
pdm.termui: Pinning: cowsay 6.1
pdm.termui: ======== Ending round 1 ========
pdm.termui: ======== Starting round 2 ========
STATUS: Resolving: new pin pip 23.3.1
pdm.termui: Pinning: pip 23.3.1
pdm.termui: ======== Ending round 2 ========
pdm.termui: ======== Starting round 3 ========
STATUS: Resolving: new pin setuptools 68.2.2
pdm.termui: Pinning: setuptools 68.2.2
pdm.termui: ======== Ending round 3 ========
pdm.termui: ======== Starting round 4 ========
STATUS: Resolving: new pin wheel 0.41.3
pdm.termui: Pinning: wheel 0.41.3
pdm.termui: ======== Ending round 4 ========
pdm.termui: ======== Starting round 5 ========
pdm.termui: ======== Resolution Result ========
pdm.termui: Stable pins:
pdm.termui:       python None
pdm.termui:       cowsay 6.1
pdm.termui:          pip 23.3.1
pdm.termui:   setuptools 68.2.2
pdm.termui:        wheel 0.41.3
STATUS: Fetching hashes for resolved packages...
pdm.termui: Fetching hashes for cowsay@6.1
pdm.termui: Fetching hashes for wheel@0.41.3
pdm.termui: Fetching hashes for pip@23.3.1
pdm.termui: Fetching hashes for setuptools@68.2.2
🔒 Lock successful
Changes are written to pyproject.toml.
STATUS: Resolving packages from lockfile...
STATUS: Fetching hashes for resolved packages...
Synchronizing working set with resolved packages: 1 to add, 3 to update, 0 to remove

pdm.termui: Removing distribution pip
unearth.preparer: Downloading <Link https://files.pythonhosted.org/packages/47/6a/453160888fab7c6a432a6e25f8afe6256d0d9f2cbd25971021da6491d899/pip-23.3.1-py3-none-any.whl (from https://pypi.org/simple/pip/)> (2.1 MB)
  ✔ Update pip 23.2.1 -> 23.3.1 successful
pdm.termui: Removing distribution setuptools
unearth.preparer: Downloading <Link https://files.pythonhosted.org/packages/bb/26/7945080113158354380a12ce26873dd6c1ebd88d47f5bc24e2c5bb38c16a/setuptools-68.2.2-py3-none-any.whl (from https://pypi.org/simple/setuptools/)> (807 kB)
Error processing line 1 of /opt/conda/lib/python3.11/site-packages/distutils-precedence.pth:

  Traceback (most recent call last):
    File "<frozen site>", line 186, in addpackage
    File "<string>", line 1, in <module>
  ModuleNotFoundError: No module named '_distutils_hack'

Remainder of file ignored
  ✔ Update setuptools 68.0.0 -> 68.2.2 successful
pdm.termui: Removing distribution wheel
unearth.preparer: Downloading <Link https://files.pythonhosted.org/packages/fa/7f/4c07234086edbce4a0a446209dc0cb08a19bb206a3ea53b2f56a403f983b/wheel-0.41.3-py3-none-any.whl (from https://pypi.org/simple/wheel/)> (65 kB)
  ✔ Update wheel 0.41.2 -> 0.41.3 successful
unearth.preparer: Downloading <Link https://files.pythonhosted.org/packages/f1/13/63c0a02c44024ee16f664e0b36eefeb22d54e93531630bd99e237986f534/cowsay-6.1-py3-none-any.whl (from https://pypi.org/simple/cowsay/)> (25 kB)
  ✔ Install cowsay 6.1 successful

🎉 All complete!
(base) root@6dda1e96f2c9:/# pdm remove -gv cowsay
Removing packages from default dependencies: cowsay
Changes are written to pyproject.toml.
STATUS: Resolving dependencies
pdm.termui: ======== Start resolving requirements ========
pdm.termui:   pip
pdm.termui:   setuptools
pdm.termui:   wheel
pdm.termui:   python>=3.11.5,<3.11.6
pdm.termui:   Adding requirement pip
pdm.termui:   Adding requirement setuptools
pdm.termui:   Adding requirement wheel
pdm.termui:   Adding requirement python>=3.11.5,<3.11.6
pdm.termui: ======== Starting round 0 ========
STATUS: Resolving: new pin python>=3.11.5,<3.11.6
pdm.termui: Pinning: python None
pdm.termui: ======== Ending round 0 ========
pdm.termui: ======== Starting round 1 ========
STATUS: Resolving: new pin pip 23.3.1
pdm.termui: Pinning: pip 23.3.1
pdm.termui: ======== Ending round 1 ========
pdm.termui: ======== Starting round 2 ========
STATUS: Resolving: new pin setuptools 68.2.2
pdm.termui: Pinning: setuptools 68.2.2
pdm.termui: ======== Ending round 2 ========
pdm.termui: ======== Starting round 3 ========
STATUS: Resolving: new pin wheel 0.41.3
pdm.termui: Pinning: wheel 0.41.3
pdm.termui: ======== Ending round 3 ========
pdm.termui: ======== Starting round 4 ========
pdm.termui: ======== Resolution Result ========
pdm.termui: Stable pins:
pdm.termui:       python None
pdm.termui:          pip 23.3.1
pdm.termui:   setuptools 68.2.2
pdm.termui:        wheel 0.41.3
STATUS: Fetching hashes for resolved packages...
🔒 Lock successful
Changes are written to pdm.lock.
STATUS: Resolving packages from lockfile...
STATUS: Fetching hashes for resolved packages...
Synchronizing working set with resolved packages: 0 to add, 0 to update, 28 to remove

pdm.termui: Removing distribution brotlipy
pdm.termui: Removing distribution certifi
pdm.termui: Removing distribution charset-normalizer
pdm.termui: Removing distribution conda-libmamba-solver
pdm.termui: Removing distribution conda-content-trust
pdm.termui: Removing distribution cffi
  ✔ Remove brotlipy 0.7.0 successful
pdm.termui: Removing distribution boltons
  ✔ Remove certifi 2023.7.22 successful
pdm.termui: Removing distribution conda-package-handling
  ✔ Remove conda-libmamba-solver 23.9.1 successful
pdm.termui: Removing distribution conda
pdm.termui: Removing distribution conda_package_streaming
  ✔ Remove conda-content-trust 0.2.0 successful
  ✔ Remove charset-normalizer 2.0.4 successful
  ✔ Remove boltons 23.0.0 successful
  ✔ Remove conda-package-handling 2.2.0 successful
pdm.termui: Removing distribution cowsay
  ✔ Remove conda-package-streaming 0.9.0 successful
pdm.termui: Removing distribution idna
pdm.termui: Removing distribution cryptography
  ✔ Remove cowsay 6.1 successful
pdm.termui: Removing distribution jsonpatch
pdm.termui: Removing distribution jsonpointer
  ✔ Remove cffi 1.15.1 successful
pdm.termui: Removing distribution libmambapy
  ✔ Remove idna 3.4 successful
pdm.termui: Removing distribution packaging
  ✔ Remove libmambapy 1.5.1 successful
pdm.termui: Removing distribution pluggy
pdm.termui: Removing distribution pycosat
  ✔ Remove pluggy 1.0.0 successful
pdm.termui: Removing distribution pycparser
  ✔ Remove jsonpointer 2.1 successful
  ✔ Remove jsonpatch 1.32 successful
  ✔ Remove packaging 23.1 successful
  ✔ Remove pycparser 2.21 successful
pdm.termui: Removing distribution pyOpenSSL
pdm.termui: Removing distribution PySocks
pdm.termui: Removing distribution requests
  ✔ Remove cryptography 41.0.3 successful
  ✔ Remove pycosat 0.6.6 successful
pdm.termui: Removing distribution ruamel.yaml
  ✔ Remove pyopenssl 23.2.0 successful
pdm.termui: Removing distribution tqdm
pdm.termui: Removing distribution truststore
pdm.termui: Removing distribution urllib3
pdm.termui: Removing distribution zstandard
  ✔ Remove requests 2.31.0 successful
  ✔ Remove truststore 0.8.0 successful
  ✔ Remove zstandard 0.19.0 successful
  ✔ Remove tqdm 4.65.0 successful
  ✔ Remove pysocks 1.7.1 successful
  ✔ Remove urllib3 1.26.16 successful
  ✔ Remove ruamel-yaml 0.17.21 successful
  ✔ Remove conda 23.9.0 successful

🎉 All complete!
(base) root@6dda1e96f2c9:/# conda
bash: /opt/conda/bin/conda: No such file or directory

```
</details>

## Actual behavior

- conda gets removed (because it is not required in the global pyproject.toml?)
- In my testing, the same happened in an empty project with non-global package installs

<details>
  <summary>Workaround to repair broken conda install</summary>

- delete the conda folder, e.g. `~/miniconda3`
- re-install conda
</details>

When installing `pdm-conda`, there is instead another issue: https://github.com/macro128/pdm-conda/issues/24.

## Expected behavior

- conda and its dependencies are somehow protected, since they are what set up the Python interpreter in the first place
- At least there should be a warning in the docs or README about interoperability problems with conda
- Relevance
  - Probably also happened here: https://github.com/pdm-project/pdm/issues/1901#issuecomment-1773170181
  - There is another person interested in combining conda/pdm: https://github.com/pdm-project/pdm/discussions/1925

## Environment Information

```
(base) root@d05172886a53:/# pdm info -g
Changes are written to pyproject.toml.
PDM version:
  2.10.1
Python Interpreter:
  /opt/conda/bin/python (3.11)
Project Root:
  /root/.config/pdm/global-project
Local Packages:
  
(base) root@d05172886a53:/# pdm info --env -g
{
  "implementation_name": "cpython",
  "implementation_version": "3.11.5",
  "os_name": "posix",
  "platform_machine": "x86_64",
  "platform_release": "5.15.0-88-generic",
  "platform_system": "Linux",
  "platform_version": "#98-Ubuntu SMP Mon Oct 2 15:18:56 UTC 2023",
  "python_full_version": "3.11.5",
  "platform_python_implementation": "CPython",
  "python_version": "3.11",
  "sys_platform": "linux"
}
```

**Comment by frostming** [comment]
The problem is PDM couldn't know which packages are reserved. However we can forbid `--global` for conda python. Is that OK?

**Comment by Firionus** [comment]
Thanks for the quick answer :rocket: :smile:. 

Unfortunately, the same thing happens when you do it non-globally in a fresh project:

<details>
  <summary>Docker Console Dump with nonglobal remove</summary>

```
(base) root@cc826050c281:/# conda --version
conda 23.9.0
(base) root@cc826050c281:/# pdm init -nv
Creating a pyproject.toml for PDM...
Project is initialized successfully
(base) root@cc826050c281:/# pdm add -v cowsay
Adding packages to default dependencies: cowsay
STATUS: Resolving dependencies
pdm.termui: ======== Start resolving requirements ========
pdm.termui:   cowsay
pdm.termui:   python>=3.11
pdm.termui:   Adding requirement cowsay
pdm.termui:   Adding requirement python>=3.11
pdm.termui: ======== Starting round 0 ========
STATUS: Resolving: new pin python>=3.11
pdm.termui: Pinning: python None
pdm.termui: ======== Ending round 0 ========
pdm.termui: ======== Starting round 1 ========
STATUS: Resolving: new pin cowsay 6.1
pdm.termui: Pinning: cowsay 6.1
pdm.termui: ======== Ending round 1 ========
pdm.termui: ======== Starting round 2 ========
pdm.termui: ======== Resolution Result ========
pdm.termui: Stable pins:
pdm.termui:   python None
pdm.termui:   cowsay 6.1
STATUS: Fetching hashes for resolved packages...
pdm.termui: Fetching hashes for cowsay@6.1
🔒 Lock successful
Changes are written to pyproject.toml.
STATUS: Resolving packages from lockfile...
STATUS: Fetching hashes for resolved packages...
Synchronizing working set with resolved packages: 1 to add, 0 to update, 0 to remove

unearth.preparer: Downloading <Link https://files.pythonhosted.org/packages/f1/13/63c0a02c44024ee16f664e0b36eefeb22d54e93531630bd99e237986f534/cowsay-6.1-py3-none-any.whl (from https://pypi.org/simple/cowsay/)> (25 kB)
  ✔ Install cowsay 6.1 successful

🎉 All complete!
(base) root@cc826050c281:/# pdm remove -v cowsay
Removing packages from default dependencies: cowsay
Changes are written to pyproject.toml.
STATUS: Resolving dependencies
pdm.termui: ======== Start resolving requirements ========
pdm.termui:   python>=3.11
pdm.termui:   Adding requirement python>=3.11
pdm.termui: ======== Starting round 0 ========
STATUS: Resolving: new pin python>=3.11
pdm.termui: Pinning: python None
pdm.termui: ======== Ending round 0 ========
pdm.termui: ======== Starting round 1 ========
pdm.termui: ======== Resolution Result ========
pdm.termui: Stable pins:
pdm.termui:   python None
STATUS: Fetching hashes for resolved packages...
🔒 Lock successful
Changes are written to pdm.lock.
STATUS: Resolving packages from lockfile...
STATUS: Fetching hashes for resolved packages...
Synchronizing working set with resolved packages: 0 to add, 0 to update, 28 to remove

pdm.termui: Removing distribution brotlipy
pdm.termui: Removing distribution certifi
pdm.termui: Removing distribution conda-content-trust
pdm.termui: Removing distribution conda-libmamba-solver
pdm.termui: Removing distribution charset-normalizer
pdm.termui: Removing distribution cffi
  ✔ Remove brotlipy 0.7.0 successful
pdm.termui: Removing distribution boltons
  ✔ Remove certifi 2023.7.22 successful
  ✔ Remove conda-libmamba-solver 23.9.1 successful
pdm.termui: Removing distribution conda-package-handling
pdm.termui: Removing distribution conda_package_streaming
pdm.termui: Removing distribution conda
  ✔ Remove conda-content-trust 0.2.0 successful
pdm.termui: Removing distribution cowsay
  ✔ Remove charset-normalizer 2.0.4 successful
  ✔ Remove boltons 23.0.0 successful
  ✔ Remove cowsay 6.1 successful
  ✔ Remove conda-package-streaming 0.9.0 successful
  ✔ Remove conda-package-handling 2.2.0 successful
pdm.termui: Removing distribution idna
pdm.termui: Removing distribution cryptography
pdm.termui: Removing distribution jsonpatch
pdm.termui: Removing distribution jsonpointer
pdm.termui: Removing distribution libmambapy
pdm.termui: Removing distribution packaging
  ✔ Remove idna 3.4 successful
  ✔ Remove cffi 1.15.1 successful
  ✔ Remove libmambapy 1.5.1 successful
pdm.termui: Removing distribution pluggy
pdm.termui: Removing distribution pycosat
  ✔ Remove packaging 23.1 successful
pdm.termui: Removing distribution pycparser
  ✔ Remove pluggy 1.0.0 successful
pdm.termui: Removing distribution pyOpenSSL
  ✔ Remove pycparser 2.21 successful
  ✔ Remove jsonpointer 2.1 successful
  ✔ Remove jsonpatch 1.32 successful
pdm.termui: Removing distribution PySocks
  ✔ Remove pyopenssl 23.2.0 successful
  ✔ Remove cryptography 41.0.3 successful
pdm.termui: Removing distribution requests
  ✔ Remove pycosat 0.6.6 successful
pdm.termui: Removing distribution ruamel.yaml
pdm.termui: Removing distribution tqdm
pdm.termui: Removing distribution truststore
pdm.termui: Removing distribution urllib3
pdm.termui: Removing distribution zstandard
  ✔ Remove truststore 0.8.0 successful
  ✔ Remove requests 2.31.0 successful
  ✔ Remove zstandard 0.19.0 successful
  ✔ Remove pysocks 1.7.1 successful
  ✔ Remove tqdm 4.65.0 successful
  ✔ Remove urllib3 1.26.16 successful
  ✔ Remove ruamel-yaml 0.17.21 successful
  ✔ Remove conda 23.9.0 successful

🎉 All complete!
(base) root@cc826050c281:/# conda --version
bash: /opt/conda/bin/conda: No such file or directory
```
</details>

**Comment by frostming** [comment]
I agree this is a bug, that it reuses the base env since PDM thinks it's already in an **activated** env.

**Comment by Firionus** [comment]
Thanks, that mitigates the global case, though I'm not sure what the effect on conda-pdm is. 

But pdm still removes conda when you remove a package from an initiated project 🫤: 

<details>
  <summary>Console Dump from Docker</summary>

```console
(base) root@92cf892ad489:/# conda --version
conda 23.9.0
(base) root@92cf892ad489:/# pdm add -g cowsay
Changes are written to pyproject.toml.
Adding packages to default dependencies: cowsay
[PdmUsageError]: Can't use global project in conda base environment since it is managed by conda
(base) root@92cf892ad489:/# pdm init -nv
Creating a pyproject.toml for PDM...
Project is initialized successfully
(base) root@92cf892ad489:/# pdm add -v cowsay
Adding packages to default dependencies: cowsay
STATUS: Resolving dependencies
pdm.termui: ======== Start resolving requirements ========
pdm.termui:   cowsay
pdm.termui:   python>=3.11
pdm.termui:   Adding requirement cowsay
pdm.termui:   Adding requirement python>=3.11
pdm.termui: ======== Starting round 0 ========
STATUS: Resolving: new pin python>=3.11
pdm.termui: Pinning: python None
pdm.termui: ======== Ending round 0 ========
pdm.termui: ======== Starting round 1 ========
STATUS: Resolving: new pin cowsay 6.1
pdm.termui: Pinning: cowsay 6.1
pdm.termui: ======== Ending round 1 ========
pdm.termui: ======== Starting round 2 ========
pdm.termui: ======== Resolution Result ========
pdm.termui: Stable pins:
pdm.termui:   python None
pdm.termui:   cowsay 6.1
STATUS: Fetching hashes for resolved packages...
pdm.termui: Fetching hashes for cowsay@6.1
🔒 Lock successful
Changes are written to pyproject.toml.
STATUS: Resolving packages from lockfile...
STATUS: Fetching hashes for resolved packages...
Synchronizing working set with resolved packages: 1 to add, 0 to update, 0 to remove

unearth.preparer: Downloading <Link https://files.pythonhosted.org/packages/f1/13/63c0a02c44024ee16f664e0b36eefeb22d54e93531630bd99e237986f534/cowsay-6.1-py3-none-any.whl (from https://pypi.org/simple/cowsay/)> (25 kB)
  ✔ Install cowsay 6.1 successful

🎉 All complete!
(base) root@92cf892ad489:/# pdm remove -v cowsay
Removing packages from default dependencies: cowsay
Changes are written to pyproject.toml.
STATUS: Resolving dependencies
pdm.termui: ======== Start resolving requirements ========
pdm.termui:   python>=3.11
pdm.termui:   Adding requirement python>=3.11
pdm.termui: ======== Starting round 0 ========
STATUS: Resolving: new pin python>=3.11
pdm.termui: Pinning: python None
pdm.termui: ======== Ending round 0 ========
pdm.termui: ======== Starting round 1 ========
pdm.termui: ======== Resolution Result ========
pdm.termui: Stable pins:
pdm.termui:   python None
STATUS: Fetching hashes for resolved packages...
🔒 Lock successful
Changes are written to pdm.lock.
STATUS: Resolving packages from lockfile...
STATUS: Fetching hashes for resolved packages...
Synchronizing working set with resolved packages: 0 to add, 0 to update, 28 to remove

pdm.termui: Removing distribution brotlipy
pdm.termui: Removing distribution certifi
pdm.termui: Removing distribution charset-normalizer
pdm.termui: Removing distribution conda-content-trust
pdm.termui: Removing distribution conda-libmamba-solver
pdm.termui: Removing distribution cffi
  ✔ Remove brotlipy 0.7.0 successful
pdm.termui: Removing distribution boltons
  ✔ Remove certifi 2023.7.22 successful
pdm.termui: Removing distribution conda-package-handling
  ✔ Remove conda-libmamba-solver 23.9.1 successful
pdm.termui: Removing distribution conda
  ✔ Remove conda-content-trust 0.2.0 successful
  ✔ Remove charset-normalizer 2.0.4 successful
pdm.termui: Removing distribution conda_package_streaming
  ✔ Remove boltons 23.0.0 successful
pdm.termui: Removing distribution cowsay
  ✔ Remove conda-package-handling 2.2.0 successful
  ✔ Remove conda-package-streaming 0.9.0 successful
pdm.termui: Removing distribution idna
  ✔ Remove cowsay 6.1 successful
pdm.termui: Removing distribution cryptography
pdm.termui: Removing distribution jsonpatch
pdm.termui: Removing distribution jsonpointer
pdm.termui: Removing distribution libmambapy
  ✔ Remove idna 3.4 successful
pdm.termui: Removing distribution packaging
  ✔ Remove libmambapy 1.5.1 successful
  ✔ Remove cffi 1.15.1 successful
pdm.termui: Removing distribution pluggy
  ✔ Remove packaging 23.1 successful
pdm.termui: Removing distribution pycosat
pdm.termui: Removing distribution pycparser
  ✔ Remove pluggy 1.0.0 successful
pdm.termui: Removing distribution pyOpenSSL
  ✔ Remove jsonpatch 1.32 successful
  ✔ Remove jsonpointer 2.1 successful
  ✔ Remove pycparser 2.21 successful
pdm.termui: Removing distribution PySocks
pdm.termui: Removing distribution requests
  ✔ Remove cryptography 41.0.3 successful
  ✔ Remove pyopenssl 23.2.0 successful
pdm.termui: Removing distribution ruamel.yaml
  ✔ Remove pycosat 0.6.6 successful
pdm.termui: Removing distribution tqdm
pdm.termui: Removing distribution truststore
pdm.termui: Removing distribution urllib3
pdm.termui: Removing distribution zstandard
  ✔ Remove requests 2.31.0 successful
  ✔ Remove truststore 0.8.0 successful
  ✔ Remove zstandard 0.19.0 successful
  ✔ Remove tqdm 4.65.0 successful
  ✔ Remove pysocks 1.7.1 successful
  ✔ Remove urllib3 1.26.16 successful
  ✔ Remove ruamel-yaml 0.17.21 successful
  ✔ Remove conda 23.9.0 successful

🎉 All complete!
(base) root@92cf892ad489:/# conda --version
bash: /opt/conda/bin/conda: No such file or directory
(base) root@92cf892ad489:/# pdm info
PDM version:
  2.10.2
Python Interpreter:
  /opt/conda/bin/python3.11 (3.11)
Project Root:
  /
Local Packages:
  
(base) root@92cf892ad489:/# pdm info --env
{
  "implementation_name": "cpython",
  "implementation_version": "3.11.5",
  "os_name": "posix",
  "platform_machine": "x86_64",
  "platform_release": "5.15.0-88-generic",
  "platform_system": "Linux",
  "platform_version": "#98-Ubuntu SMP Mon Oct 2 15:18:56 UTC 2023",
  "python_full_version": "3.11.5",
  "platform_python_implementation": "CPython",
  "python_version": "3.11",
  "sys_platform": "linux"
}
```
</details>

Poetry does the same thing when you run `poetry install -sync`. 

Is this fixable at all, or should the docs just advise against combining conda with pdm unless you use pdm-conda?

**Comment by frostming** [comment]
Here is the correct output you expect to see when running `pdm init`:
```
$ pdm init -n
Creating a pyproject.toml for PDM...
Virtualenv is created successfully at /Users/fming/wkspace/playground/test-pdm/.venv
Project is initialized successfully
```
But if `/opt/conda/bin/python3.11` is already saved in `.pdm-python`, there is no way for PDM to switch to other interpreter without human interrupt.

**Comment by frostming** [comment]
There was one place that has not been fixed. Try 2.10.3

**Comment by Firionus** [comment]
I can confirm this is fixed in v2.10.4, conda isn't removed anymore :+1:
